src/live.asp.net/Services/FeedService.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace live.asp.net.Services
+{
+    public class FeedService
+    {
+        private readonly IShowsService _showsService;
+        private readonly bool _disableCache;
+
+        public FeedService(IShowsService showsService, bool? disableCache)
+        {
+            _showsService = showsService;
+            _disableCache = disableCache ?? false;
+        }
+
+        public async Task<string> GetFeed()
+        {
+            var parent = new StringWriter();
+            var settings = new XmlWriterSettings { Indent = true };
```

Comment:
My idea is to use POCO, `Rss`, `Channel` and `Item` classes after that we can use the `XmlSerializer` to serialize the `Rss` instance